src/app/constants.py

```diff
@@ -0,0 +1,51 @@
+from django.utils.safestring import mark_safe
+
+# License submission form help text
+
+FULL_NAME_HELP_TEXT = mark_safe('\u2022 The full name may omit certain word, such as \'the,\' for alphabetical sorting purposes. <br/>'
```

Comment:
Change `\'the,\'` to the

Reply:
Please review other lines too.

Reply:
Ok

Reply:
@rtgdk You really do not want single quotes around such words? Please have a look at the image below, you can read and see the use of those quotes. I will remove them if it is still your wish after this.
![ek1](https://user-images.githubusercontent.com/45539225/82905400-c38f2500-9f5b-11ea-9ef3-ba82a2859646.png)

Reply:
@ekongobie My bad, I didn't understand the usage of the backslash and quote.